bootstrap/terraform/README.md

```diff
@@ -147,6 +150,10 @@ echo "$(kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.
 ## Clean up
 1. Delete resources created by Crossplane such as first Claims, then XRDs and Compositions.
 
+1. Remove crossplane providers by setting `enable = false` in main.tf for each provider and running `terraform apply`
```

Comment:
This kind of things should be in variables. I think there are other things that should be exposed in variables so let's re-visit. Please open an issue to track.

Reply:
https://github.com/awslabs/crossplane-on-eks/issues/147